packages/edit-post/src/components/layout/index.js

```diff
@@ -124,7 +131,9 @@ function useEditorStyles() {
 }
 
 function Layout() {
+	useCommands();
```

Comment:
We already have the hook in edit post [here](https://github.com/WordPress/gutenberg/blob/trunk/packages/edit-post/src/editor.js#L32).

Reply:
I can remove it from the Editor component and leave it where I initially added it in the Layout component. This will make sure things stay consistent between edit-post and edit-site packages.

Reply:
Done in 5e9662994463d561964c2355c3887a868878f480.